pkgs/data/themes/andromeda-gtk-theme/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+stdenvNoCC.mkDerivation {
+  pname = "andromeda-gtk-theme";
+  version = "250751a";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "Andromeda-gtk";
+    rev = "250751a546dd0fa2e67eef86d957fbf993b61dfe";
+    sha256 = "0k86c4jm7mw9b2rg7rmg0rbbklh2nw63v50j0rfgzn6nz27zs6kv";
+  };
```

```suggestion
  };

  propagatedUserEnvPkgs = [
    gtk-engine-murrine
  ];
```
Don't know if this is needed since I can't find anything about it in [the manual](https://nixos.org/manual/nixpkgs/unstable/#ssec-gnome-packaging). A lot of packages use it though.

---

Should be good to include this since the theme includes a GTK2 variant with [`engine "murrine"`](https://github.com/EliverLara/Andromeda-gtk/blob/250751a546dd0fa2e67eef86d957fbf993b61dfe/gtk-2.0/main.rc#L2521-L2528)